keyboards/keyhive/maypad/info.json

```
@@ -0,0 +1,12 @@
+{
+    "keyboard_name": "maypad", 
+    "url": "https://keyhive.xyz/shop/may-pad", 
+    "maintainer": "codybender", 
+    "width": 4, 
+    "height": 5, 
+    "layouts": {
+        "LAYOUT": {
```

This key needs to be an exact match for the macro name from `maypad.h`.

```suggestion
        "LAYOUT_numpad_5x4": {
```